components/ILIAS/Authentication/classes/class.ilAuthLoginPageEditorGUI.php

```diff
@@ -133,15 +149,24 @@ public function executeCommand(): void
         }
     }
 
-
     /**
      * Forward to page editor
      */
     protected function forwardToPageObject(): void
     {
+        $keys = $this->http->wrapper()->query()->retrieve(
+            'loginpage_languages_key',
+            $this->refinery->byTrying([
+                $this->refinery->kindlyTo()->listOf($this->refinery->kindlyTo()->string()),
+                $this->refinery->always([])
+            ])
+        );
+
+        $this->key = ilLanguage::lookupId(current($keys));
```

Comment:
What happens, if `$keys` is an empty array? Maybe we should make this more robust.

Reply:
If $keys is an empty array the lookupId() returns a "0". I would suggest to check that return value "0" and if so, show a  setOnScreenMessage like "The selected language does not exist".  Is that ok?

Reply:
Applying `current` on an empty array evaluates in `false`, not `0`. IMO this will cause an error when passed to `\ilLanguage::lookupId`, which expects a `string`.

Reply:
ok.